src/components/githubRepoCard/GithubRepoCard.js

```diff
@@ -25,8 +25,8 @@ export default function GithubRepoCard({ repo }) {
         <div className="repo-stats">
           <div className="repo-left-stat">
             <span>
-              <div className="language-color" style={{ backgroundColor: repo.node.primaryLanguage.color }}></div>
-              <p>{repo.node.primaryLanguage.name}</p>
+              <div className="language-color" style={{ backgroundColor: repo.node.primaryLanguage === null ? "black" : repo.node.primaryLanguage.color}}></div>
+              <p>{repo.node.primaryLanguage === null ? "Undefined" : repo.node.primaryLanguage.name}</p>
```

Comment:
If there is no primary language found just don't show the primary language don't need to show **Undefined** or black dot this would look a little weird.

Reply:
Hide the language div if no primary language is found.

Reply:
I modified the code according to saadpasta's comment.